src/api/routes/v1/validator.ts

```diff
@@ -42,6 +42,10 @@ interface ValidatorResponse {
   } | null
   partial: boolean
   unl: boolean
+  amendments?: string | Array<{ id: string; name: string }>
```

Comment:
You should remove `string` being allowed.  I dont think there is much use for it to be a string.